### PR TITLE
feat: feature branch-helper

### PR DESCRIPTION
**Pull request by l0kal**

### CODE REVIEW

**Comment by sasjo** [comment]
I wonder about the action name? I think branch-helper is rather broad. Could something like branch-info be more clear to show the intent of the action?

File: branch-helper/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "build": "ncc build src/index.js",
+    "watch": "watch 'npm run build' ./src"
+  },
+  "dependencies": {
+    "@actions/core": "^1.2.0",
+    "@actions/exec": "^1.0.1",
+    "@actions/github": "^2.0.1"
```
Comment (sasjo) [inline branch-helper/package.json]: Remove unused dependencies.

Comment (l0kal) [reply branch-helper/package.json]: They should be removed now.

File: branch-info/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "branch-helper",
```
Comment (sasjo) [inline branch-info/package.json]: Rename the package to branch-info

Comment (l0kal) [reply branch-info/package.json]: fixed

**Pull request merged**